lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -1,14 +1,16 @@
 from datetime import datetime
 from http import HTTPStatus
-from logging import Logger
-from typing import Any, Callable, Dict
+from logging import Logger, error
+from typing import Any, Callable, Dict, Iterator
```

Comment:
Note that type hinting a generator with `Generator[...]` is ... weird. PEP says that `Iterator[root-type]` is a valid equivalent, and much easier to read.